cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -12,6 +12,8 @@ metadata:
   name: rook-ceph
 allowPrivilegedContainer: true
 allowHostDirVolumePlugin: true
+allowHostNetwork: false # set to true if running rook with the provider as host
```

Comment:
it's preferred to have comments on the previous line, and a suggestion on the text...
```suggestion
# set to true if running rook with host networking enabled
allowHostNetwork: false 
```